nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/test/java/org/nuxeo/ecm/platform/picture/core/test/TestPictureRenditions.java

```diff
@@ -112,4 +117,25 @@ public void shouldExposeOnlyMarkedPictureViewsAsRenditions() throws Exception {
         runtimeHarness.undeployContrib("org.nuxeo.ecm.platform.picture.core",
                 "OSGI-INF/imaging-picture-renditions-override.xml");
     }
+
+    @Test
+    public void shouldDeclareRenditionDefinitionImageToPDF() {
+        List<RenditionDefinition> renditionDefinitions = renditionService.getDeclaredRenditionDefinitions();
+        List<RenditionDefinition> imageToPDFRenditionDefinitions = renditionDefinitions.stream()
+            .filter(rD -> rD.getName().equals("imageToPDF")).collect(Collectors.toList());
+        assertEquals(1, imageToPDFRenditionDefinitions.size());
+        RenditionDefinition imageToPDFRenditionDefinition = imageToPDFRenditionDefinitions.get(0);
+        assertEquals("Image.Blob.ConvertToPDF", imageToPDFRenditionDefinition.getOperationChain());
+        assertEquals(1, imageToPDFRenditionDefinition.getFilterIds().size());
+        assertEquals("hasPictureFacet", imageToPDFRenditionDefinition.getFilterIds().get(0));
+    }
+
+    @Test
+    public void shouldMakeRenditionAvailableImageToPDF() throws Exception {
+        Blob source = Blobs.createBlob(FileUtils.getResourceFileFromContext("images/test.jpg"));
+        DocumentModel doc = session.createDocumentModel("/", "picture", "Picture");
+        doc.setProperty("file", "filename", source.getFilename());
+        doc.setProperty("file", "content", source);
+        assertNotNull(renditionService.getRendition(doc, "imageToPDF"));
```

Comment:
Seems that this test does not test what it should if my previous comment is true about the `convert` vs `converter` parameter name.